packages/server/src/config.ts

```diff
@@ -111,6 +111,31 @@ function baseUrlFromEnv(env: EnvVariables, appPort: number): string {
 	}
 }
 
+function ldapConfigFromEnv(env: EnvVariables): LdapConfig[] {
+	return [
+		{
+			enabled: env.LDAP_1_ENABLED,
+			userCreation: env.LDAP_1_CREATE_USER,
+			host: env.LDAP_1_SERVER,
+			mailAttribute: env.LDAP_1_MAIL_ATTRIBUTE,
+			fullNameAttribute: env.LDAP_1_FULLNAME_ATTRIBUTE,
+			baseDN: env.LDAP_1_BASE_DN,
+			bindDN: env.LDAP_1_BIND_DN,
+			bindPW: env.LDAP_1_BIND_PW,
+		},
+		{
+			enabled: env.LDAP_2_ENABLED,
+			userCreation: env.LDAP_2_CREATE_USER,
```

Comment:
I think there should be a check to see if LDAP_1_ENABLED and LDAP_2_ENABLED are defined. If they are not, you'd get an empty array (and the code should handle this). If only one of them is defined, you get an array of one element, etc.